tutor4points/core/templates/points.html

```diff
@@ -9,26 +9,27 @@
 {% block heading %}
 Points
 <font color="red">
-{% if error_message %}<p><strong>{{ error_message }}</strong></p>{% endif %}
+    {% if error_message %}<p><strong>{{ error_message }}</strong></p>{% endif %}
```

Comment:
I think we could remove this part since we are already embedding the errors to the form.